app/views/admin/tasks/payroll_details.html.erb

```diff
@@ -48,7 +48,11 @@
           <fieldset class="govuk-fieldset govuk-!-margin-bottom-6">
             <legend class="govuk-fieldset__legend govuk-fieldset__legend--l ">
               <h3 class="govuk-heading-m">
-                <%= I18n.t("#{@claim.policy.to_s.underscore}.admin.task_questions.payroll_details.title", bank_or_building_society: I18n.t("admin.#{@claim.bank_or_building_society}")) %>
+                <%= I18n.t(
+                  "#{@claim.policy.to_s.underscore}.admin.task_questions.payroll_details.title",
+                  bank_or_building_society: I18n.t("admin.#{@claim.bank_or_building_society}"),
+                  claimant_name: @claim.full_name
+                ) %>
```

Comment:
```suggestion
                <%= I18n.t(
                  "admin.task_questions.payroll_details.title",
                  bank_or_building_society: I18n.t("admin.#{@claim.bank_or_building_society}"),
                  claimant_name: @claim.full_name
                ) %>
```
could this just use a single `admin.task_questions.payroll_details.title`, as they're the same for all journeys now?